packages/web-client/src/helpers/share/role.ts

```diff
@@ -409,6 +409,12 @@ export abstract class LinkShareRoles {
       return bitmask === (bitmask | r.bitmask(false))
     })
   }
+
+  static getByName(name: string, isFolder: boolean): ShareRole {
+    return LinkShareRoles.list(isFolder).find(
```

Comment:
We need the other capability checks here, no? Because alias (internal) links are set to `false` by default, so `list()` will ignore those roles.